docs/requirements.txt

```diff
@@ -1,4 +1,4 @@
-conda-sphinx-theme==0.1.1
+conda-sphinx-theme==0.1.3
```

Comment:
In conda we don't pin at all, should that change?

https://github.com/conda/conda/blob/d891d9f59f193a3e28bf28c959f41b08e304f31c/docs/requirements.txt#L1